cmd/bosun/web/static/js/0-bosun.ts

```diff
@@ -78,6 +78,21 @@ bosunApp.config(['$routeProvider', '$locationProvider', function($routeProvider:
 		otherwise({
 			redirectTo: '/',
 		});
+	$httpProvider.interceptors.push(function($q) {
```

Comment:
so this wraps all angular-originated api calls with the miniprofiler flag? What calls will come from a different context that we don't want miniprofiler on?